src/components/modular-content/responsive-image/responsive-image.njk

```diff
@@ -0,0 +1,24 @@
+{% macro responsiveImage(blockData, context) %}
+  {% set image = blockData.image %}
+
+  <div class="responsive-image container {% if context.modifiers.large %}container--large {% else %}container--small{% endif %}">
+    <figure
+      class="responsive-image__figure {% if context.modifiers.rounded %}responsive-image__figure--rounded{% endif %}"
+      style="padding-top: {{ 100.0 / image.responsiveImage.aspectRatio }}%;
+             background-image: url({{ image.responsiveImage.base64 }});
+             background-size: cover;"
+    >
+      <picture class="responsive-image__picture">
+        <source type="image/webp" srcset="{{ image.url | imageUrl({ w: 776, fm: 'webp' }) }}">
+        <img
+          alt="{{ image.alt }}"
+          src="{{ image.url | imageUrl({ w: 776 }) }}"
```

Comment:
where does 776 comes from?

Reply:
It's something from standard responsive image Voorhoede Dato practices